docs/docs/react-without-jsx.md

```diff
@@ -4,11 +4,13 @@ title: React Without JSX
 permalink: docs/react-without-jsx.html
 ---
 
-JSX is not a requirement for using React. Using React without JSX is especially convenient when you don't want to set up compilation in your build environment.
+JSX is not a requirement for using React. React exposes an API for creating elements with functions instead of JSX. This is especially convenient if you want to use React without introducing compilation steps to your build process.
```

Comment:
This is a little misleading since JSX is itself converted to createElement calls.